spec/player_config_spec.rb

```diff
@@ -0,0 +1,78 @@
+require_relative '../lib/player_config.rb'
+require_relative '../lib/human_player.rb'
+require_relative '../lib/computer_player.rb'
+
+describe PlayerConfig do
+
+  let(:user_interface_double) { double("user interface") }
```

Comment:
Using `instance_double` may allow you to cut down on the number of `allow().to receive()` calls you currently need.

Reply:
Seemed instance_double didn't do the trick once I added them and started deleting allow/receive lines.  Instead I put the common allow/receive calls in a before(:each).